aten/src/ATen/LegacyBatchingRegistrations.cpp

```diff
@@ -588,6 +588,17 @@ bool _has_same_storage_numel_batching_rule(const Tensor& self, const Tensor& oth
   return true;
 }
 
+bool _chunk_grad_outputs_efficient_attention_batch_rule(
```

Comment:
As you've noticed, this file doesn't actually get used by our vmap implementation anymore. It's a relic from the past that we have yet to delete.

Reply:
oops i meant to delete this!